.github/workflows/rentzila_tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Rentzila Playwright Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Install Playwright browsers
+        run: npx playwright install
+
+
+      - name: Set up environment variables
+        run: |
+          echo "ADMIN_EMAIL=${{secrets.ADMIN_EMAIL}}" >> .env
+          echo "ADMIN_PASSWORD=${{secrets.ADMIN_PASSWORD}}" >> .env
+          echo "VALID_EMAIL=${{ secrets.VALID_EMAIL}}" >> .env
+          echo "VALID_PASSWORD=${{ secrets.VALID_PASSWORD}}" >> .env
+          echo "VALID_PHONE=${{ secrets.VALID_PHONE}}" >> .env
+          echo "CORRECT_PHONE_NUMBERS_FULL=${{ secrets.CORRECT_PHONE_NUMBERS_FULL }}" >> .env
+          echo "CORRECT_PHONE_NUMBERS_WITHOUT_PLUS=${{ secrets.CORRECT_PHONE_NUMBERS_WITHOUT_PLUS }}" >> .env
+          echo "CORRECT_PHONE_NUMBERS_WITHOUT_PLUS38=${{ secrets.CORRECT_PHONE_NUMBERS_WITHOUT_PLUS38 }}" >> .env
+          echo "HOMEPAGE_URL=${{ secrets.HOMEPAGE_URL }}" >> .env
+
+      - name: Run tests
+        run: npm run test:chrome
+
+      - name: Upload test results
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: test-results
+          path: test-results/
+
+      - name: Upload videos
```

Comment:
That is nice but also deploy the report to a separate branch and enable the GitHub Pages (write to me if you need help)

Reply:
I didn't deploy the report on GitHub Pages, because It is optional for this project. And that was said on our meetings